api/src/pcapi/models/feature.py

```diff
@@ -123,6 +123,7 @@ class FeatureToggle(enum.Enum):
     WIP_SUGGESTED_SUBCATEGORIES = "Activer les sous-catégories suggérées par IA lors de la création d'offre"
     ENABLE_COLLECTIVE_OFFERS_EXPIRATION = "Activer la mise en avant des offres collectives sur le point d'expirer"
     ENABLE_COLLECTIVE_NEW_STATUSES = "Activer les nouveaux statuts des offres collectives"
+    ENABLE_COLLECTIVE_DRAFT_OFFERS = "Activer les brouillons pour les offres collectives" 
```

Comment:
si c'est un ff que vous allez supprimer faut le préfixer par "WIP"

`# For features under construction, a temporary feature flag must be named with the `WIP_` prefix`